syntax = "proto3";

package courseservice;

// Сообщение запроса
message GetCourseRequest {
    string course_id = 1;
}

// Сообщение ответа
message GetCourseResponse {
    string course_id = 1;
    string title = 2;
    string description = 3;
}

//Определяем gRRC сервис
service CourseService {
    rpc GetCourse (GetCourseRequest) returns (GetCourseResponse);
}
